dashboard/app/models/teacher_feedback.rb

```diff
@@ -80,12 +80,16 @@ def self.get_latest_feedbacks_received(student_id, level_id, script_id)
 
   # returns the latest feedback for each student on every level in a script given by the teacher
   def self.get_latest_feedbacks_given(student_ids, level_ids, script_id, teacher_id)
+    query = {
```

Comment:
I think might be helpful to add in the header comment which of the input arguments can be nil and what that means?

Reply:
I've added a full comment block describing params, now I'm wondering, should I have this full block for every method in teacher_feedback? Is this too much?